<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edURL"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ключ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edKey"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>От:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edFrom"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Кому:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edTo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Тема:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edSubject"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Сообщение:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="txtMessage"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bnSend">
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
